### PR TITLE
Add snifftest vcxproj file and documentation

### PR DESCRIPTION
**Pull request by lealem47**
# Description

The snifftest won't build by default and will have to be manually enabled, so as to not break existing builds (because of dependencies). Any ideas on how to better handle this are welcome.

# Testing

Building and running on windows.

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [ ] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

File: sslSniffer/sslSnifferTest/include.am [hunk 1]
```diff
@@ -12,4 +12,5 @@ EXTRA_DIST += sslSniffer/README.md
 EXTRA_DIST += sslSniffer/sslSniffer.vcproj
 EXTRA_DIST += sslSniffer/sslSniffer.vcxproj
 EXTRA_DIST += sslSniffer/sslSnifferTest/sslSniffTest.vcproj
+EXTRA_DIST += sslSniffer/sslSnifferTest/sslSniffTest.vcxproj
```
Comment (dgarske) [inline sslSniffer/sslSnifferTest/include.am:15]: You’ll need to add the new README_WIN.md here also.

**Review by dgarske — CHANGES_REQUESTED** [review]
Otherwise looks good

File: sslSniffer/sslSnifferTest/include.am (hunk 1 above)
Comment (lealem47) [reply sslSniffer/sslSnifferTest/include.am:15]: Done.

File: sslSniffer/sslSnifferTest/README_WIN.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- Next, download the WinPcap development pack found [here](https://www.winpcap.org/devel.htm).
+
+- Extract the `WpdPack` folder from the downloaded zip and place adjacently to the wolfSSL directory as shown below.
+
+        Projects\
+            wolfssl\
+            WpdPack\
+
+- Then on Visual Studios, open the configuration manager for the wolfssl solution. This can be done by right-clicking on the solution viewer then selecting `properties`. The button for the configuration manager should be on the top-right of the window.
```
Comment (haydenroche5) [inline sslSniffer/sslSnifferTest/README_WIN.md:13]: Visual Studios -> Visual Studio

**Review by haydenroche5 — CHANGES_REQUESTED** [review]
Just one typo. Once fixed, I'll merge.

File: sslSniffer/sslSnifferTest/README_WIN.md (hunk 2 above)
Comment (lealem47) [reply sslSniffer/sslSnifferTest/README_WIN.md]: Fixed.

**Review by haydenroche5 — APPROVED** [review]

**Review by dgarske — APPROVED** [review]

**Pull request merged**